components/Templates/Templates.php

```diff
@@ -395,6 +395,23 @@ public function add_meta_boxes() {
 					'restrict_capability' => true,
 				),
 			),
+			array(
+				'name'       => 'show_restrict_message',
+				'label'      => __( 'Show no access message?', 'pods' ),
+				'default'    => 1,
+				'type'       => 'boolean',
+				'dependency' => true,
+			),
+			array(
+				'name'                  => 'restrict_message',
+				'label'                 => __( 'No access message', 'pods' ),
+				'type'                  => 'wysiwyg',
+				'default'               => __( 'You do not have access to view this content.', 'pods' ),
+				'wysiwyg_editor_height' => 200,
+				'depends-on'            => array(
+					'show_restrict_message' => true,
+				),
+			)
```

Comment:
* :no_entry_sign: Comma required after last value in array declaration
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 5a3482489271bae52396a1592e6a6371</sub>